client-app/shared/catalog/components/product-card-grid.vue

```diff
@@ -100,7 +100,7 @@
         <template #trigger>
           <router-link
             :to="link"
-            :target="target"
+            :target="browserTarget"
```

Comment:
why we can't use $cfg.details_browser_target here?

Reply:
To keep only one logic entry